OpenQA/Isotovideo/CommandHandler.pm

```diff
@@ -350,4 +351,37 @@ sub update_status_file ($self) {
     rename $tmp, AUTOINST_STATUSFILE or die $!;
 }
 
+my ($last_check_seconds, $last_check_microseconds) = gettimeofday;
+sub _calc_check_delta ($self) {
+    # an estimate of eternity
+    my $delta = $last_check_seconds ? tv_interval([$last_check_seconds, $last_check_microseconds], [gettimeofday]) : 100;
+    # sleep the remains of one second if $delta > 0
+    my $timeout = $delta > 0 ? 1 - $delta : 0;
+    $self->timeout($timeout < 0 ? 0 : $timeout);
+    return $delta;
+}
+
+sub check_asserted_screen ($self, $testfd) {
```

Comment:
`$testfd` is always the same, right? So maybe it makes sense to make it an attribute of CommandHandler and pass it when instantiating the object like the oder FDs. Then it would be more consistent with how the CommandHandler works in general.